package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "img-magnifier-extension",
+  "version": "1.0.0",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "build": "node build.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/dhruvdhaduk-simform/img-magnifier-extension.git"
+  },
+  "keywords": [],
+  "author": "",
```

Can add a proper keywords and author(your) name:
```suggestion
  "keywords": ["image", "magnifier", "extension", "zoom", "browser"],
  "author": "Dhruv Dhaduk",
```

---

Added keywords and author.
https://github.com/dhruvdhaduk-simform/img-magnifier-extension/pull/1/commits/564cf393c8f6b659c85d8609710d7e9c55c2c0c0